docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,195 @@
+       service/my-trino-cluster   ClusterIP   10.96.25.35   <none>        8080/TCP   161m
+
+       NAME                                           READY   UP-TO-DATE   AVAILABLE   AGE
+       deployment.apps/my-trino-cluster-coordinator   1/1     1            1           161m
+       deployment.apps/my-trino-cluster-worker        2/2     2            2           161m
+
+       NAME                                                     DESIRED   CURRENT   READY   AGE
+       replicaset.apps/my-trino-cluster-coordinator-bfb74c98d   1         1         1       161m
+       replicaset.apps/my-trino-cluster-worker-76f6bf54d6       2         2         2       161m
+
+Executing queries
+-----------------
+
+The pods running the Trino containers are all running on a private network
+internal to Kubernetes. In order to access them, specifically the coordinator,
+you will need to create a tunnel to the exposed service and your computer. You
+can do this by running the commands generated upon installation.
+
+#. Store the coordinator pod name in a shell variable called ``POD_NAME``.
+
+   .. code-block:: text
+
+       export POD_NAME=$(kubectl get pods --namespace default -l "app=trino,release=my-trino-cluster,component=coordinator" -o jsonpath="{.items[0].metadata.name}")
+
+#. Create the tunnel from the service to the client. After you run this you can
+   connect to the Trino coordinator at ``http://localhost:8080``.
+
+   .. code-block:: text
+
+       kubectl port-forward $POD_NAME 8080:8080
+
+#. To connect to Trino, you can use the :doc:`/client/cli` or your favorite
+   client that supports using a :doc:`/client/jdbc`.
+   :ref:`Install the command-line interface <cli-installation>`, and connect to
+   Trino.
+
+   .. code-block:: text
+
+       ./trino --server http://localhost:8080
+
+#. Using the generated benchmark data in the ``tpch`` catalog, type and execute
+   a query on the ``nation`` table using the scale factor schema, ``sf1``. Try
+   other commands to explore this data set:
+
+   .. code-block:: text
+
+       $ trino> select count(*) from tpch.sf1.nation;
+        _col0
+       -------
+         25
+       (1 row)
+
+       Query 20181105_001601_00002_e6r6y, FINISHED, 1 node
+       Splits: 21 total, 21 done (100.00%)
+       0:06 [25 rows, 0B] [4 rows/s, 0B/s]
+
+#. Once you are done with your exploration, enter the ``quit`` command.
+
+
+Configuring Helm for Trino
+--------------------------
+
+The :doc:`container image </installation/containers>` already contains a
+default configuration to get started, and some catalogs to allow you to explore
+Trino. You can customize these configuration by updating the ``values.yaml``
+file.
+
+
+The ``values.yaml`` has various properties that, when not empty, will create
+configuration files in the ``/etc/trino`` directory structure as created in the
+:doc:`/installation/deployment`. One common configuration request is to add
```

Comment:
This assumes the reader has any idea that Trino uses configuration files. Maybe we shouldn't, at least at the beginning of this section. Linking to the deployment might be a useful reference but at the end of this section.

Reply:
I'm confused about how moving it to the end of the section helps. I imagine linking this where it is, actually sheds light on how Trino is configured if they so desire to click.

Reply:
My point is that when using the chart you don't mount config files anywhere but put them inside values.yaml. If we're not explicit about it, it can be confusing for people new to Trino.

Reply:
Yeah, but we should be clear that the values added to the values.yaml correlates to configuration files that are installed on each Trino node. Anywhere else they look in the docs we will be referring to configuration properties files and that could get confusing if they don't know the basic installation method. So that's why I point them there to make sure they get educated.

I do see your point though, so maybe it would make sense to just clarify more. Maybe adding a paragraph to say something like:
> Kubernetes allows you to mimic a [traditional deployment](https://trino.io/docs/current/installation/deployment.html) by updating the `values.yaml` file. It's important to understand how files such as the Trino configuration, JVM, and various catalog properties are configured in Trino before continuing.

Then after that, we can assume the reader will know to look for this before getting to this section. WDYT?

Reply:
Sounds great!

Reply:
Yeah ... maybe just rephrase .. the values file is just a template and helm creates the config files from it on the containers